### PR TITLE
Adds Vulpkanin Nerve Pinch

### PR DESCRIPTION
**Pull request by QuinnAggeler**
Gives Vulpkanin a maneuver to pacify an irrational being by gripping a nerve cluster at the base of their neck.
It also only works on Vulpkanin.

This is only logical

:cl:HugoLuman
rscadd: Adds the Vulpkanin Nerve Pinch maneuver
/:cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
isn't this mostly for cats

**Comment by Spacemanspark** [comment]
Is this a meme PR
And yes. It's mostly for cats.

**Comment by sasanek12** [comment]
Maymays everywhere

**Comment by FlattestGuitar** [comment]
Waht.

**Comment by tigercat2000** [comment]
http://niceme.me

**Comment by fludd12** [comment]
relocate proc to tajarans, make it work by picking up other tajarans with your mouth.

cats lol

**Comment by Iamgoofball** [comment]
what is this retarded shit

time for me to dissect it like every other shitty furry PR

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -167,8 +167,41 @@
...
+	set category = "Abilities"
+	set name = "Nerve pinch"
+	set desc = "Incapacitate someone who's being completely illogical by pinching a nerve cluster on their neck"
+
+	if(stat)
+		to_chat(src, "You cannot perform the maneuver in this state.")
+		return
+
+	var/list/choices = list()
+	for(var/mob/living/carbon/human/H in view(1,src))
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: >>>view()

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 2]
```diff
@@ -167,8 +167,41 @@
...
+	if(!M || !src) return
+
+	if(!(src.Adjacent(M))) return
+
+	to_chat(src, "<span class='notice'>You pinch a pressure point at the base of [M]'s neck.</span>")
+	visible_message("<span class='warning'>[usr] pinches the base of [M]'s neck!</span>")
+	to_chat(M, "<span class='warning'><b>HRK!</b></span>")
+	M.Weaken(5)
+	M.Stun(5)
+	M.stat = UNCONSCIOUS
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: use Paralyse(5)

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 3]
```diff
@@ -167,8 +167,41 @@
...
+	var/mob/living/carbon/human/M = input(src,"Who is being too illogical?") in null|choices
+
+	if(!M || !src) return
+
+	if(!(src.Adjacent(M))) return
+
+	to_chat(src, "<span class='notice'>You pinch a pressure point at the base of [M]'s neck.</span>")
+	visible_message("<span class='warning'>[usr] pinches the base of [M]'s neck!</span>")
+	to_chat(M, "<span class='warning'><b>HRK!</b></span>")
+	M.Weaken(5)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: replace all 3 of these with Paralyse(5)

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 4]
```diff
@@ -167,8 +167,41 @@
...
+	var/list/choices = list()
+	for(var/mob/living/carbon/human/H in view(1,src))
+		var/species = H.get_species()
+		var/race = get_species()
+		if(!H.stat && species == race && src.Adjacent(H) && src != H)
+			choices += H
+
+	var/mob/living/carbon/human/M = input(src,"Who is being too illogical?") in null|choices
+
+	if(!M || !src) return
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: same line returns are a nogo

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 5]
```diff
@@ -167,8 +167,41 @@
...
+
+	if(!(src.Adjacent(M))) return
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: same line returns are a nogo

**Comment by TheDZD** [comment]
# No.

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 6]
```diff
@@ -167,8 +167,41 @@
...
+		return
+
+	var/list/choices = list()
+	for(var/mob/living/carbon/human/H in view(1,src))
+		var/species = H.get_species()
+		var/race = get_species()
+		if(!H.stat && species == race && src.Adjacent(H) && src != H)
+			choices += H
+
+	var/mob/living/carbon/human/M = input(src,"Who is being too illogical?") in null|choices
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: gee you know what this lets me do

this lets me walk next to someone, hit the button, and move the prompt to the side

then I can OPEN PALM SLAM THAT OK BUTTON right as I run by them and get a 5 second knockout stun.

Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: it also doesn't have any kind of a cooldown

Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: and it works on stunned people so I can already just spam this on someone and perma knockout them

**Comment by NTSAM** [comment]
👽 ✌️ 
live long and
   meme

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 7]
```diff
@@ -167,8 +167,41 @@
...
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/nerve_pinch
+		)
+
 /datum/species/vulpkanin/handle_death(var/mob/living/carbon/human/H)
 	H.stop_tail_wagging(1)
+	
+/mob/living/carbon/human/proc/nerve_pinch()
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: >proc verb

try action buttons

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 8]
```diff
@@ -167,8 +167,41 @@
...
+	set category = "Abilities"
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: action button, verbs overcrowding

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 9]
```diff
@@ -167,8 +167,41 @@
...
+	set name = "Nerve pinch"
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: *Nerve Pinch

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 10]
```diff
@@ -167,8 +167,41 @@
...
+	set desc = "Incapacitate someone who's being completely illogical by pinching a nerve cluster on their neck"
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: the vulkan joke doesn't work and is really fucking forced and shoehorned

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 11]
```diff
@@ -167,8 +167,41 @@
...
+
+	if(stat)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: i can use this WHILE STUNNED

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 12]
```diff
@@ -167,8 +167,41 @@
...
+		to_chat(src, "You cannot perform the maneuver in this state.")
+		return
+
+	var/list/choices = list()
+	for(var/mob/living/carbon/human/H in view(1,src))
+		var/species = H.get_species()
+		var/race = get_species()
+		if(!H.stat && species == race && src.Adjacent(H) && src != H)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species/station.dm]: why does this only work on dogbeest

**Comment by Iamgoofball** [comment]
@NTSAM did you miss the comment where I point out the joke is shit

**Comment by NTSAM** [comment]
calm down and take a pun PR christ sake

**Comment by tigercat2000** [comment]
@Iamgoofball Knock it off. You're being way too hostile for a meme PR.

**Comment by Iamgoofball** [comment]
no fun allowed

File: code/modules/mob/living/carbon/human/species/station.dm (hunk 6 above)
Comment (QuinnAggeler) [inline code/modules/mob/living/carbon/human/species/station.dm]: You're right, that is way too short of a stun

**Comment by Regen1** [comment]
>no fun allowed

You're the one huffing and screaming at everyone's face bby, calm down or you'll loose good boy points

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 13]
```diff
@@ -167,8 +167,39 @@
...
+	var/mob/living/carbon/human/M = input(src,"Who is being too illogical?") in null|choices
+
+	if(!M || !src) return
+
+	if(!(src.Adjacent(M))) return
+
+	to_chat(src, "<span class='notice'>You pinch a pressure point at the base of [M]'s neck.</span>")
+	visible_message("<span class='warning'>[usr] pinches the base of [M]'s neck!</span>")
+	to_chat(M, "<span class='warning'><b>HRK!</b></span>")
+	M.Paralyse(2700)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/station.dm]: Too short of a stun; this only lasts 90 minutes; please increase to 4500 to account for extra long shifts and shuttle arrival time.

**Comment by QuinnAggeler** [comment]
@Fox-McCloud I'm sorry Captain, but I cannae extend the time annae further! 132 minutes is all she's got! Unless we could find some sort o' three part special...

**Comment by Iamgoofball** [comment]
pfffffff

**Comment by taukausanake** [comment]
I like how he's not being violent at the PR but rather helping to make the code better. Even if just to screw with the furry alien race

**Comment by KasparoVy** [comment]
Lmfao wtf, I'm just here for the keks

**Comment by Dumbdumn5** [comment]
I have no idea what the hell is wrong with everything above.

**Comment by Vivalas** [comment]
I was wrong for assuming Github was comprised of mostly mature and sensible people.

**Comment by Dumbdumn5** [comment]
@Vivalas 

> I was wrong for assuming "any part of the SS13 community"* was comprised of mostly mature and sensible people.

**Comment by Fox-McCloud** [comment]
Merging right away.

**Comment by QuinnAggeler** [comment]
Awesome, now just for the Unathi Bat'leth

**Pull request closed without merging**